test/runtests.jl

```diff
@@ -53,4 +54,10 @@ Logging.with_logger(hide_logs ? Logging.NullLogger() : Logging.current_logger())
     end
 end
 
+if logfile !== nothing
+    open(logfile, "w") do io
+        write(io, Pkg.DEFAULT_IO[])
+    end
+end
```

Comment:
huh, I didn't know this kind of thing worked.. Doesn't `Pkg.DEFAULT_IO[]` get consumed before this?

Reply:
It doesn't look like it's consumed by anything :)